postgres/datadog_checks/postgres/postgres.py

```diff
@@ -320,6 +321,9 @@ def dynamic_queries(self):
             queries.append(SUBSCRIPTION_STATE_METRICS)
         if self.version >= V15:
             queries.append(STAT_SUBSCRIPTION_STATS_METRICS)
+        if self.version >= V16:
+            if self._config.dbm_enabled:
```

Comment:
do we want this to be exclusively for DBM?

Reply:
> do we want this to be exclusively for DBM?

@ynnadkrap [DBMON-3520](https://datadoghq.atlassian.net/browse/DBMON-3520) says to only support this for DBM initially. The metrics seem like something we'd want to open up to all eventually. Happy to adjust as we see fit.  I see that most DBM specific metric collecting happens in it's own carved out metric collection groups. I had put these new metrics where they seemed to make the most sense in the short term for just DBM and longer term of opening up for all.